### ISSUE TITLE
Should we close stale issues?

### ISSUE THREAD

**Issue by elijahr2411** [codehz/ElementZero#173]
Our open issues page is full of issues that haven't had any activity for months. I propose that to keep our open issues page just issues that need attention, we close all open issues that haven't had any activity for, lets say, a month. What are your thoughts on this?

**Comment by Weissnix4711** [comment]
Nah. We just need people to actually look through, and reply to them. I honestly can't be bothered.
